model_compression_toolkit/core/pytorch/default_framework_info.py

```diff
@@ -95,4 +105,4 @@
                                      ACTIVATION2MINMAX,
                                      LAYER2MINMAX,
                                      KERNEL_ATTRIBUTES,
-                                     OUTPUT_CHANNEL_INDEX)
```

Comment:
Please remove the variable OUTPUT_CHANNEL_INDEX if it is not being used anywhere else

Reply:
same